Automattic/vip-go-mu-plugins
# Remove sneaky .DS_Store file and fix .gitignore to ignore globally

## Description
.DS_Store is a hidden data file native to MacOS and can have [security concerns](https://buildthis.com/ds_store-files-and-why-you-should-know-about-them/) when it is put in places where it should not be.

> Back in 2015, a .DS_Store file was exploited and used to gain access to an admin portal of the TCL Corporation, a multinational Chinese electronics company. The entire backend and database of the application were exposed to anyone that accessed the .DS_Store file.

- Removed .DS_Store hiding in search/es-wp-query/tests
- Changed .gitignore to ignore this file globally.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1. Check out PR on a MacOS machine
1. Navigate into mi-plugins sub directories with finder app
1. verify that there is no .DS_Store files in uncommitted changes with `git status`
